### PR TITLE
Don't clean the before-deploy task...

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by sbrunner** [comment]
Can I merge it?

File: .travis.yml [hunk 1]
```diff
@@ -32,18 +32,9 @@ before_script:
...
-- provider: pypi
-  server: https://upload.pypi.org/legacy/
-  user: sbrunner
-  password:
-    secure: "PoIuVu4sdApkWm6CXnBMMzXCj1vNnEdLXKqcBkXlKjBxh6JlivGRmqoFizJAe3KNjZMwMBMcJkMR+f53g0z3P06sIx9WQCaAjYV0j5w5Si71tjjOWohY2DjIaUcNNOsodZc9KDdtSpN0Xm4+vbn7Lg9FofCovs2KAIsBONVD/I4="
-  distributions: sdist bdist_wheel
-  skip_upload_docs: true
-  skip_cleanup: true
+- provider: script
+  script: travis/deploy-daily-pypi
```
Comment (arnaud-morvan) [inline .travis.yml]: Seems that this do not match with new script name after renaming.

Comment (sbrunner) [reply .travis.yml]: Oups, thanks, fixed :-)

**Comment by arnaud-morvan** [comment]
Yes, feel free to merge.

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**